data/settings.js

```diff
@@ -10453,4 +10453,451 @@ If enabled, protocol-level SSL errors are logged. Same as
+changetype: modify
+replace: olcAccess
+olcAccess: {0}to attrs=userPassword
+	by self write
+	by dn="cn=authuser,dc=test,dc=dovecot,dc=net" read
+	by * auth
+olcAccess: {1}to *
+	by self read
+	by dn="cn=authuser,dc=test,dc=dovecot,dc=net" read
+	by * auth
+\`\`\`
+:::
+
+2. Run \`ldapmodify\` to apply the change.
+
+\`\`\`console
+$ ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f doveauth_access.ldif
+\`\`\``
+	},
+
+	passdb_ldap_bind_userdn: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+If authentication binding is used, you can save one LDAP request per login
+if users' DN can be specified with a common template. The template can use
+the standard %variables (see [\`user_filter\`](#user-filter)).
+
+Note that you can't use any \`fields\` declaration if you use this setting.
+
+Example: \`passdb_ldap_bind_userdn = cn=%u,ou=people,o=org\``
+	},
+
+	ldap_base: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+LDAP base.
+
+[[variable]] can be used.
+
+Example: \`ldap_base = dc=mail, dc=example, dc=org\``
+	},
+
+	use_worker: {
+		tags: [ 'ldap' ],
+		default: 'no',
+		values: setting_types.BOOLEAN,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+By default all LDAP lookups are performed by the auth master process.
+
+If \`use_worker=yes\`, auth worker processes are used to perform the lookups.
+
+Each auth worker process creates its own LDAP connection so this can
+increase parallelism.
+
+With \`use_worker=no\`, the auth master process can keep 8 requests pipelined
+for the LDAP connection, while with \`use_worker=yes\` each connection has a
+maximum of 1 request running.
+
+For small systems, \`use_worker=no\` is sufficient and uses less resources.`
+	},
+
+	ldap_debug_level: {
+		tags: [ 'ldap' ],
+		default: '0',
+		values: setting_types.UINT,
+		text: `
+LDAP library debug level as specified by \`LDAP_DEBUG_*\` in \`ldap_log.h\`.
+
+Value \`-1\` means everything.
+
+You may need to recompile OpenLDAP with debugging enabled to get enough
+output.`
+	},
+
+	default_pass_scheme: {
+		tags: [ 'ldap' ],
+		default: 'crypt',
+		values: setting_types.STRING,
+		text: `
+::: info
+	LDAP Authentication Only
+:::
+
+Default password scheme. \`{scheme}\` before password overrides this.
+
+See [[link,password_schemes]] for a list of supported schemes.`
+	},
+
+	ldap_deref: {
+		tags: [ 'ldap' ],
+		default: 'never',
+		values: setting_types.ENUM,
+		values_enum: [ 'never', 'searching', 'finding', 'always' ],
+		text: `
+Specify dereference which is set as an LDAP option.`
+	},
+
+	ldap_auth_dn: {
+		tags: [ 'ldap' ],
+		default: '<empty>',
+		values: setting_types.STRING,
+		text: `
+Specify the Distinguished Name (the username used to login to the LDAP
+server).
+
+Leave it commented out to bind anonymously (useful with
+\`passdb_ldap_bind = yes\`).
+
+Example: \`ldap_auth_dn = uid=dov-read,dc=ocn,dc=ad,dc=jp,dc=.\``
```

Comment:
What's with the `dc=.` at the end, is that really valid? Also, would be better to use example.com domains.